framework/include/mfem/containers/MFEMContainers.h

```diff
@@ -0,0 +1,194 @@
+#ifdef MFEM_ENABLED
+
+#pragma once
+#include <map>
+#include <set>
+#include <string>
+#include <memory>
+#include <vector>
+#include "libmesh/ignore_warnings.h"
+#include <mfem.hpp>
+#include "libmesh/restore_warnings.h"
+
+namespace platypus
+{
+
+/// Lightweight adaptor over an std::map from strings to pointer to T
+template <typename T>
```

Comment:
Lots of camel case methods here. 

Is it it not possible to have moose context here to make errors more rich?

Reply:
I could pass through the MFEMProblem to the MFEMProblemData to the EquationSystem which owns these maps to get to `mooseError`, but getting an error from the problem doesn't give much information regarding the input file